index.html

```diff
@@ -7,162 +7,215 @@
+
+<!--        <label for="email">Email</label> <br/>-->
+<!--        <input type="email" id="email" name="email" required/> <br/>-->
+
+<!--        <label for="password">Password</label> <br/>-->
+<!--        <input type="password" id="password" name="password" required> <br/>-->
+
+<!--        <label for="range">Your programming level</label> <br/>-->
+<!--        <input type="range" id="range" name="range" min="1" max="10"> <br/>-->
+<!--        <p>-->
+<!--            What does HTML stand for? <br/>-->
+<!--            <input type="checkbox" id="hyperlinks" name="html" value="Hyperlinks and Text Markup Language">-->
+<!--            <label for="hyperlinks">Hyperlinks and Text Markup Language</label> <br>-->
+
+<!--            <input type="checkbox" id="home_tool" name="html" value="Home Tool Markup Language">-->
+<!--            <label for="home_tool">Home Tool Markup Language</label> <br>-->
+
+<!--            <input type="checkbox" id="html" name="html" value="Hyper Text Markup Language">-->
+<!--            <label for="html">Hyper Text Markup Language</label> <br>-->
+
+<!--        </p>-->
+<!--        <p>-->
+<!--            What does HTTP stand for? <br/>-->
+<!--            <input type="radio" id="translate" name="http" value="Hyper Text Translate Protocol">-->
+<!--            <label for="translate">Hyper Text Translate Protocol</label> <br>-->
+
+<!--            <input type="radio" id="transmission" name="http" value="Hyper Text Transmission Protocol">-->
+<!--            <label for="transmission">Hyper Text Transmission Protocol</label> <br>-->
+
+<!--            <input type="radio" id="transfer" name="http" value="Hyper Text Transfer Protocol">-->
+<!--            <label for="transfer">Hyper Text Transfer Protocol</label> <br>-->
+
+<!--        </p>-->
+<!--        <p>-->
+<!--            <label for="comment">Comment</label> <br/>-->
+<!--            <textarea id="comment" name="comment" cols="30" rows="10" maxlength="500">Comment</textarea>-->
+
+<!--        </p>-->
+<!--        <label for="html_stand_for">What does HTML stand for?</label> <br/>-->
+<!--        <select id="html_stand_for" name="html stand for">-->
+<!--            <option disabled selected>Your answer</option>-->
+<!--            <option value="1">Hyper Text Translate Protocol</option>-->
+<!--            <option value="2">Hyper Text Transmission Protocol</option>-->
+<!--            <option value="3">Hyper Text Transfer Protocol</option>-->
+<!--        </select>-->
+<!--        <p>-->
+<!--            <button type="submit">Send</button>-->
+<!--        </p>-->
+<!--    </fieldset>-->
+<!--</form>-->
+
+<div class="section">
+
+    <div class="title">
+        <h2>What do we offer</h2>
+    </div>
+    <div class="under-title">
+        <img src="images/line.png" alt="Just a line">
+    </div>
+
+    <div class="cards">
+        <div class="card">
+            <div class="icon">
```

Comment:
Don't add wrappers if you can escape them
for example you could use only `<img> `here
```
img {
    width: 100px;
    height: 100px;
    object-fit: contain;
}
```
[Read more](https://www.w3schools.com/css/css3_object-fit.asp)